extensions/BMO/template/en/default/global/choose-product.html.tmpl

```diff
@@ -98,14 +98,6 @@
       icon="component.png"
     %]
     [% INCLUDE easyproduct
-      name="Firefox OS"
-      icon="firefox_os.png"
-    %]
-    [% INCLUDE easyproduct
-      name="Marketplace"
-      icon="marketplace.png"
-    %]
-    [% INCLUDE easyproduct
       name="Thunderbird"
```

Comment:
I don't really see how Thunderbird and Seamonkey are comparable here, the projects you are removing are not being actively developed (not sure about the fxos community). AIUI the current stance on Thunderbird/Seamonkey is that they are tolerated as long as they don't stand in the way, and I don't think a link on the enter_bug form is standing in the way.